packages/cycle-scripts/scripts/eject.js

```diff
@@ -29,21 +29,28 @@ inquirer.prompt([ejectConfirmation]).then(answers => {
   // STEP 1 - Prepare package.json
   // Declaring new scripts
   const scripts = {
-    start: 'node scripts/start.js',
-    test: 'node scripts/test.js',
-    build: 'node scripts/build.js'
+    start: 'NODE_ENV=development webpack-dev-server --config webpack.config.js',
```

Comment:
I don't think that this is the right approach we should use for putting things into PR and deciding what should be a feature and what's not, there are issue to discuss this...